data.sql

```diff
@@ -1,5 +1,22 @@
+-- Day 1
+
 insert into animals(name,date_of_birth, weight_kg,neutered,escape_attempts) values ('Agumon','2020-02-03', 10.23,true,0);
 insert into animals(name,date_of_birth, weight_kg,neutered,escape_attempts) values 
   ('Gabumon','2018-11-15', 8,true,2),
   ('Pikachu','2021-01-07',15.04,false,1),
   ('Devimon','2017-05-12', 11,true,5);
+
+-- Day 2
+
+alter table animals add column species varchar(100) ;
```

Comment:
- [ ] **_Everything that creates or changes the table structure should be saved in the schema.sql file._** This command here adds a specie column to the `animals` table, and this in effect changes the table, therefore, it should be placed in the `schema.sql` file. Kindly fix this